source/macos/AdaptiveCards/AdaptiveCards/Views/ACRSingleLineInputTextView.swift

```diff
@@ -175,17 +176,9 @@ extension ACRSingleLineInputTextView: InputHandlingViewProtocol {
         return self.textView.isRequired
     }
     
-    weak var errorDelegate: InputHandlingViewErrorDelegate? {
-        get {
-            return self.textView.errorDelegate
-        }
-        set {
-            self.textView.errorDelegate = newValue
-        }
-    }
-    
     func showError() {
         self.textView.showError()
+        errorDelegate?.inputHandlingViewShouldShowError(self)
```

Comment:
you will need to manage hide error as well

Reply:
Hide error will handle by ACRView.